laborator/content/toolchain-decompilare/demo/goto/goto.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Author: Gabriel Mocanu <gabi.mocanu98@gmail.com>
+ */
+
+#include <stdio.h>
+
+int main() {
+
+    int nr = 1;
+    int i = 0;
+
+loop:
+
+    if(i < 10) {
```

Comment:
Nit: format if correctly
```suggestion
    if (i < 10) {
```